DOC-3132: On chrome, comment card would be jumping when editing a ver… (#3605)
tinymce/tinymce-docs: modules/ROOT/pages/7.7.0-release-notes.adoc

For information on the **Image Optimizer** plugin, see: xref:uploadcare.adoc[Image Optimizer].

=== Comments

The {productname} {release-version} release includes an accompanying release of the **Comments** premium plugin.

**Comments** includes the following fixes.

> ==== On chrome, comment card would be jumping when editing a very large comment.
> // #TINY-11729
> 
> Previously, when editing a comment larger than the height of the comments sidebar, the sidebar would briefly jump before returning to its intended position. This caused visual disruptions that impacted the editing experience.  
> 
> {productname} {release-version} resolves this issue by ensuring the comment sidebar remains stable, preventing unintended jumps when editing large comments. 
> 

Reviewer: Normally i wouldn't change the title, but in this case its required
```suggestion
==== Comment card jumps when editing a large comment in Chrome
// #TINY-11729

Previously, when editing a comment larger than the height of the comments sidebar, the sidebar would briefly jump before returning to its intended position. This caused visual disruptions that impacted the editing experience.  

{productname} {release-version} resolves this issue by ensuring the comment sidebar remains stable, preventing unintended jumps when editing large comments. 

```

> {productname} {release-version} resolves this issue by improving the logic that handles the resizing of the comment textarea, ensuring that the editor no longer jumps when editing large comments.

Reviewer 2: I don't thinking mentioning the logic that handles resizing is necessary for the customers

> Previously, an issue was identified where, when editing comments larger than the height of the comments sidebar, the sidebar would jump up before returning to its intended position. This caused UI glitches that impacted the user experience.
> 
> {productname} {release-version} resolves this issue by improving the logic that handles the resizing of the comment textarea, ensuring that the editor no longer jumps when editing large comments.

Reviewer: ```suggestion
Previously, when editing a comment larger than the height of the comments sidebar, the sidebar would briefly jump before returning to its intended position. This caused visual disruptions that impacted the editing experience.  

{productname} {release-version} resolves this issue by ensuring the comment sidebar remains stable, preventing unintended jumps when editing large comments. 
```
